subgraphs/uniswap-v3-forks/src/common/utils/backfill.ts

```diff
@@ -64,6 +69,42 @@ export function populateEmptyPools(event: ethereum.Event): void {
     pool.totalLiquidity = poolContract.liquidity();
     pool.activeLiquidity = pool.totalLiquidity;
 
+    const tickSpacing = poolContract.tickSpacing();
+
+    const maxTick: number = customCeil(887272 / tickSpacing) * tickSpacing;
+    const minTick: number = -maxTick;
+    const ticksPerWord: number = 256 * tickSpacing;
+    const wordsCount: number = customCeil((maxTick - minTick) / ticksPerWord);
+
+    let totalLiquidityGross = BIGINT_ZERO;
+    for (let i = 0; i < wordsCount; i++) {
+      const tickBitmapIndex = customFloor(minTick / ticksPerWord) + i;
+      const populatedTicks = tickLensContract.getPopulatedTicksInWord(
+        poolAddress,
+        tickBitmapIndex as i32
+      );
+
+      for (let j = 0; j < populatedTicks.length; j++) {
+        const populatedTick = populatedTicks[j];
+        const tick = getOrCreateTick(
+          event,
+          pool,
+          BigInt.fromI32(populatedTick.tick as i32)
+        );
+        tick.liquidityGross = populatedTick.liquidityGross;
+        tick.liquidityNet = populatedTick.liquidityNet;
+        tick.lastUpdateTimestamp = event.block.timestamp;
+        tick.lastUpdateBlockNumber = event.block.number;
+        tick.save();
+
+        totalLiquidityGross = totalLiquidityGross.plus(
+          populatedTick.liquidityGross
+        );
+      }
+    }
```

Comment:
Can you add somewhere in here a link to some docs where someone could look to understand what's going on in here?

To know what 887272 means, tickBitmap, words, etc

Reply:
Sure! I added some docs and a comment. Let me know if that works.